packages/benchmarks/src/nlPromptResponse/loadTechSupportDataset.ts

```diff
@@ -0,0 +1,111 @@
+import fs from "fs";
+import path from "path";
+import { parse } from "csv-parse/sync";
+import { NlPromptResponseEvalCase } from "./NlQuestionAnswerEval";
+import { LanguageModel } from "ai";
+import {
+  classifyMongoDbMetadata,
+  tagifyMetadata,
+} from "mongodb-rag-core/mongoDbMetadata";
+
+/**
+  Interface representing a row in the tech support Q&A CSV file
+ */
+export interface TechSupportQARow {
+  /** The question asked by the user */
+  Question: string;
+  /** The answer provided to the question */
+  Answer: string;
+  /** The third column which contains validation status ("True - Correct" or "False - Incorrect") */
+  "": string;
```

Comment:
Is this supposed to be empty string?

Reply:
yea, that's how it is in their spreadsheet